### PR TITLE
[SM-839] remove enable from copy

### PR DESCRIPTION
**Pull request by willmartian**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Removes copy with "enable" from the SM bulk access dialog

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/locales/af/messages.json
Comment (vincentsalucci) [inline apps/web/src/locales/af/messages.json:1]: These changes look fine but I don't believe they should be made on the `.../af/messages.json` file. Instead these should be reverted and made to the '.../en/messages.json' file.

Comment (willmartian) [reply apps/web/src/locales/af/messages.json:1]: 😮‍💨 Thank you

File: apps/web/src/locales/af/messages.json [hunk 1]
```diff
@@ -6627,7 +6627,7 @@
     "message": "Secrets Manager"
   },
   "secretsManagerAccessDesc": {
```
Comment (vincentsalucci) [inline apps/web/src/locales/af/messages.json:6629]: This message signature needs to change as well to properly waterfall into the crowd-in translations. Could just extend `...Desc` to `...Description` 👍

Comment (willmartian) [reply apps/web/src/locales/af/messages.json]: Done. Thanks!

**Review by vincentsalucci — APPROVED** [review]

**Pull request merged**